src/LibVLCSharp.WPF/ForegroundWindow.cs

```diff
@@ -84,8 +85,12 @@ void Background_Loaded(object sender, RoutedEventArgs e)
             Owner = _wndhost;
 
             _wndhost.Closing += Wndhost_Closing;
-            _bckgnd.SizeChanged += Wndhost_SizeChanged;
+            if (!Debugger.IsAttached)
```

Comment:
Why is this needed? why not always `on` to avoid a hard-to-spot behavior change depending on whether the debugger is attached?

Reply:
you're right. We would create a tricky case. And its only a minimal loss in terms of performance. So it would be ok without.